arraycontext/container/arithmetic.py

```diff
@@ -142,6 +143,11 @@ def with_container_arithmetic(
 
     :arg bcast_number: If *True*, numbers broadcast over the container
         (with the container as the 'outer' structure).
+    :arg bcast_actx_array_type: If *True*, instances of base array types of the
+        container's array context are broadcasted over the container. Can be
+        *True* iff the container has *_cls_has_array_context_attr* set.
```

Comment:
```suggestion
        *True* only if the container has *_cls_has_array_context_attr* set.
```
? To avoid the (technical?) contraction.

Reply:
Sure, thanks!